frontend/src/createStream/SwipeableSteps/__tests__/SelectDeviceSwipeableSteps.test.jsx

```diff
@@ -83,6 +83,7 @@ describe("<SelectDeviceSwipeableSteps/> class component", () => {
       const props = components.at(0).props();
 
       const expected = {
+        className:"halfScreenVertically",
```

Comment:
Should have a space after `className` (linter is complaining about it)

Reply:
727be76